yjit/src/codegen.rs

```diff
@@ -1030,6 +1031,36 @@ fn gen_putobject(jit: &mut JITState, ctx: &mut Context, cb: &mut CodeBlock, ocb:
     KeepCompiling
 }
 
+fn gen_putself(jit: &mut JITState, ctx: &mut Context, cb: &mut CodeBlock, ocb: &mut OutlinedCb) -> CodegenStatus
+{
+    // Load self from CFP
+    let cf_opnd:X86Opnd = mem_opnd((8 * SIZEOF_VALUE) as u8, REG_CFP, RUBY_OFFSET_CFP_SELF);
```

Comment:
Detail but I don't think you need a type annotation on cf_opnd here?

Reply:
Removed the unneeded annotations.